docs/how-to-guides/SPICE/creating-spice-config-files-in-spiceql.md

```diff
@@ -0,0 +1,253 @@
+
+**Conf file for Messenger, $ISISDATA/messenger/kernels/ck/kernels.0001.conf** 
+
+```
+  Object = Instrument
+  
+    Group = Selection
+      Match = ("Instrument","InstrumentId","MDIS-WAC")
+      File = ("messenger", "kernels/ck/mdis_kernels.????.db")
+      File = ("messenger", "kernels/ck/messenger_mdis_att_kernels.????.db")
+      File = ("messenger", "kernels/ck/messenger_kernels.????.db")
+    End_Group
+  
+    Group = Selection
+      Match = ("Instrument","InstrumentId","MDIS-NAC")
+      File = ("messenger", "kernels/ck/mdis_kernels.????.db")
+      File = ("messenger", "kernels/ck/messenger_mdis_att_kernels.????.db")
+      File = ("messenger", "kernels/ck/messenger_kernels.????.db")
+  End_Group
+  
+  End_Object
+  End
+```
+**makedb file for Messenger, $ISISDATA/messenger/kernels/ck/kernels.0001.conf** 
+
+```
+
+  #  Generate kernel database from existing kernels
+
+  kerneldbgen to='messenger_kernels.????.db' type=CK \
+              recondir='$messenger/kernels/ck' \
+              reconfilter='msgr_????_v??.bc' \
+              smitheddir='$messenger/kernels/ck' \
+              smithedfilter='msgr_mdis_sc????_usgs_v?.bc' \
+              sclk='$messenger/kernels/sclk/messenger_????.tsc' \
+              lsk='$base/kernels/lsk/naif????.tls'
+              
+  kerneldbgen to='messenger_mdis_att_kernels.????.db' type=CK \
+              recondir='$messenger/kernels/ck' \
+              reconfilter='msgr_mdis_sc??????_??????_sub_v?.bc' \
+              sclk='$messenger/kernels/sclk/messenger_????.tsc' \
+              lsk='$base/kernels/lsk/naif????.tls'
+  
+  kerneldbgen to='mdis_kernels.????.db' type=CK \
+              recondir='$messenger/kernels/ck' \
+              reconfilter='msgr_mdis_gm??????_??????v?.bc' \
+              sclk='$messenger/kernels/sclk/messenger_????.tsc' \
+              lsk='$base/kernels/lsk/naif????.tls'
+
+```
+
+Here, you can the how dependencies on other CK kernels are expressed. As MDIS-WAC and MDIS-NAC have the same dependencies, the two groups are the same. 
+
+Everything here, except kernel start and stop times, requires a knowledgeable individual to manually label kernel qualities and dependencies. Times are cached by kerneldbgen.
+
+## Translating .db/makedb files to a single configuration file 
+
+!!! Warning
+    As this library is a work in progress, much of this is subject to change. With so many missions with different requirements, this format may miss supporting specific requirements for some missions. If you have any suggested changes for this process, feel free to open an issue and start a discussion. 
+
+**Example Configuration File:**
+
+Here, we'll run through an example config file, again for Messenger, encapsulating a lot of the information put into ISIS between the .db and makedb files.  
+
+.. literalinclude:: ../../SpiceQL/db/mess.json 
```

Comment:
Not sure if this is rendering properly, shows lines 209-211 together on one line?